js/ui/paygUnlockDialog.js

```diff
@@ -328,6 +333,53 @@ var PaygUnlockDialog = new Lang.Class({
         }
     },
 
+    _onSuccess: function(mainBox) {
+            mainBox.remove_all_children();
+
+            //Measuring the ceiling of the timeRemainingSecs because by the time this runs, some time has already elapsed
+            let timeString = Main.paygManager._convertTimeToString(Math.ceil(Main.paygManager._timeRemainingSecs()));
+            let successLabel = new St.Label({ style_class: 'unlock-dialog-payg-title',
+                                        text: _(timeString + " have been added to your PayGo credit."),
```

Comment:
You can't use string concatenation in a translated string, you'd need to use something like:

```js
_("%s have been added to your PayGo credit.", timeString)
```

but this won't work because "1 minute have been added to your PayGo credit" is not correct English (it's "1 minute has" vs "2 minutes have"). Other languages have different plurals. Gettext does support this but I think you're actually going to need to move this string into `timeToString` to do that.